tests/aggregation/tests.py

```diff
@@ -1325,3 +1326,55 @@ def test_aggregation_random_ordering(self):
             ('Stuart Russell', 1),
             ('Peter Norvig', 2),
         ], lambda a: (a.name, a.contact_count), ordered=False)
+
+    def test_aggregation_default_unsupported_by_count(self):
+        msg = 'Count does not support the default argument.'
+        with self.assertRaisesMessage(ValueError, msg):
+            Author.objects.aggregate(count=Count('age', default=0))
```

Comment:
No QuerySet needed, right?

```suggestion
            Count('age', default=0)
```